src/main/java/com/country/name/NameSearchRequest.java

```diff
@@ -1,10 +1,12 @@
 package com.country.name;
 
+import java.beans.ConstructorProperties;
+
 public class NameSearchRequest {
     private String startsWith;
     private String endsWith;
     private String contains;
-
+@ConstructorProperties({"prefix","suffix","contains"})
     public NameSearchRequest(String startsWith, String endsWith, String contains) {
```

Comment:
インデント揃えたいのと、カンマのあとに半角スペースいれたいですね！
```suggestion
    @ConstructorProperties({"prefix", "suffix", "contains"})
    public NameSearchRequest(String startsWith, String endsWith, String contains) {
```

Reply:
actions on save がOFFになっているかもしれません。